Tools/AP_Periph/can.cpp

```diff
@@ -1083,6 +1089,130 @@ void AP_Periph_FW::can_baro_update(void)
 #endif // HAL_PERIPH_ENABLE_BARO
 }
 
+
+/*
+  update CAN airspeed
+ */
+void AP_Periph_FW::can_airspeed_update(void)
+{
+#ifdef HAL_PERIPH_ENABLE_AIRSPEED
+    if (!airspeed.healthy()) {
+        static uint32_t last_probe_ms;
+        uint32_t now = AP_HAL::millis();
+        if (now - last_probe_ms >= 1000) {
+            last_probe_ms = now;
+            airspeed.init();
+        }
+    }
+    airspeed.update(false);
+    if (last_airspeed_update_ms == airspeed.last_update_ms()) {
+        return;
+    }
+
+    last_airspeed_update_ms = airspeed.last_update_ms();
+    if (!airspeed.healthy()) {
+        // don't send any data
+        return;
+    }
+    const float press = airspeed.get_differential_pressure();
+    float temp;
+    if (!airspeed.get_temperature(temp)) {
+        temp = nanf("");
+    } else {
+        temp += C_TO_KELVIN;
+    }
+
+    uavcan_equipment_air_data_RawAirData pkt {};
+    pkt.differential_pressure = press;
+    pkt.static_air_temperature = temp;
```

Comment:
This should be the differential air_temperature.

Reply:
it's weird, have a look at our airspeed can driver